Realm/ObjectServerTests/RealmServer.swift

```diff
@@ -809,6 +817,41 @@ public class RealmServer: NSObject {
     @objc public func createApp() throws -> AppId {
         try createAppForBSONType("string")
     }
+
+    // Retrieve MongoDB Realm AppId with ClientAppId using the Admin API
+    public func retrieveAppId(clientAppId: String) throws -> String {
```

Comment:
This concept should not be exposed as a public method. `removeUserForApp` should take the clientAppId as that is the only concept that should be exposed to the non-server caller.